src/Resources/config/routing/sonata_admin.xml

```diff
@@ -6,7 +6,7 @@
         <default key="permanent">true</default>
     </route>
     <route id="sonata_admin_dashboard" path="/dashboard">
-        <default key="_controller">Sonata\AdminBundle\Action\DashboardAction</default>
+        <default key="_controller">sonata.admin.action.dashboard</default>
```

Comment:
Keep FQNC is the better option and is used by [Symfony](https://symfony.com/doc/4.4/routing.html).

Reply:
This is true for Symfony Apps, but not for standalone bundles. Please read: https://github.com/sonata-project/SonataAdminBundle/pull/6210#discussion_r463934870  (it was discussed in this same PR)

Reply:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<routes xmlns="http://symfony.com/schema/routing" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/routing https://symfony.com/schema/routing/routing-1.0.xsd">
    <!-- do not work -->
    <import resource="sonata.user.controller.api.user" type="xml" name-prefix="sonata_api_user_user_"/>
    <!-- work -->
    <import resource="Sonata\UserBundle\Controller\Api\UserController" type="annotation" name-prefix="sonata_api_user_user_"/>
</routes>
```
I working on something simmilar with @phansys. Using service id as resource is not possible. That why I ask about FQNC. Good to know using it in controller is available. What about?:
```suggestion
        <route id="sonata_admin_dashboard" path="/dashboard" controller="sonata.admin.action.dashboard">
```